docs/sdkx-react-native/sdk-configuration.mdx

```diff
@@ -139,6 +141,29 @@ showFAQsWithConfig({
 - Provide tier-based support by setting enableContactUs to `ALWAYS` for paid users and `AFTER_VIEWING_FAQS` for unpaid ones.
 - Provide country-based support by setting enableContactUs to `ALWAYS` for local users and `AFTER_VIEWING_FAQS` for foreign ones.
 
+## Conversation prefill text {#conversation-prefill-text}
+
+<ConversationPrefillTextDescription />
+
+<div className="compact ">
+
+|        |                           |
+| ------ | ------------------------- |
+| Flag   | `conversationPrefillText` |
+| values | Any string value          |
+
+</div>
+
+Example:
+
+```javascript
+let installConfig = {
+  conversationPrefillText: "",
```

Comment:
maybe provide an example string here for prefill

Reply:
Added a prompt for inserting the prefill text (same as other books)